mongodb-odbc-driver/bin/atlas-integration-test-success-cases.yml

```diff
@@ -34,12 +34,12 @@
       - DEFAULT_AUTH=mysql_clear_password
       - ENABLE_CLEARTEXT_PLUGIN=1
 
```

Comment:
These tests just don't work anymore, and it's related to settings on Atlas.

Reply:
Could you file a build failure ticket for investigating & re-enabling them?